src/test/java/bitbot/ToDosTest.java

```diff
@@ -0,0 +1,32 @@
+package bitbot;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class ToDosTest {
+
+    Task toDos = new ToDos("Do Homework");
+
+    @Test
```

Comment:
I like that the test method names are clear and descriptive, which helps in understanding the test's purpose.